app/views/families/index.html.erb

```diff
@@ -1,6 +1,25 @@
-<p class="title"><%= t(".title") %></p>
-<%= link_to t(".add_family"), new_family_path, class: "link_button, button" %>
-<table>
-  <%= render "families_table_header" %>
-  <%= render @families %>
-</table>
+<%= render "sub_header" %>
+<div id="active_families">
+  <p class="title"><%= t(".active_families") %></p>
+  <%= link_to t(".add_family"), new_family_path, class: "link_button, button" %>
```

Comment:
Does this link belong inside of this `div`?

Reply:
Yes, I did it in purpose because I want the `add family` button only in the page of the active families. ☺️